pkgs/applications/graphics/hydrus/default.nix

```diff
@@ -0,0 +1,112 @@
+{ stdenv
+, fetchFromGitHub
+, lzma
+, qt5
+, wrapQtAppsHook
+, miniupnpc_2
+, swftools
+, pythonPackages
+, python
```

Comment:
I would import one or the other, but not both.

you can do `python.pkgs` to get the packageset
or do `pythonPackages.python` to get the interpreter